app-ios/Sources/SponsorFeature/SponsorView.swift

```diff
@@ -21,20 +82,143 @@ public struct SponsorView: View {
+                                columns: 2
+                            )
+                            .onTapItem { sponsor in
+                                viewStore.send(.showLinkSheet(sponsor))
+                            }
+                            Divider().padding(.horizontal, 16)
+                            SponsorGridView(
+                                title: "SPONSORS",
+                                sponsors: viewStore.sponsors.filter { $0.plan == .supporter },
+                                columns: 2
+                            )
+                            .onTapItem { sponsor in
+                                viewStore.send(.showLinkSheet(sponsor))
+                            }
+                        }
+                        .padding(.horizontal, 16)
+                        .padding(.vertical, 24)
+                    }
+                    .sheet(item: viewStore.binding(get: { $0.sheetItem }, send: .hideLinkSheet)) { item in
+                        if let url = URL(string: item.sponsor.link) {
+                            SafariView(url: url)
+                        }
+                    }
+                }
+            }
+            .task {
+                viewStore.send(.refresh)
+            }
+            .frame(minWidth: 0, maxWidth: .infinity)
+            .frame(minHeight: 0, maxHeight: .infinity)
+            .background(AssetColors.background.swiftUIColor)
+            .foregroundColor(AssetColors.onBackground.swiftUIColor)
+            .navigationTitle(L10n.Sponsor.title)
+            .navigationBarTitleDisplayMode(.inline)
+        }
+    }
+}
+
+struct SponsorGridView: View {
+
+    let title: String
+    let sponsors: [Sponsor]
+    let columns: Int
+    var action: (Sponsor) -> Void = { _ in }
+
+    var body: some View {
+        VStack(alignment: .leading) {
+            Text(title)
+                .font(.system(size: 22, weight: .semibold, design: .default))
+            LazyVGrid(columns: Array(repeating: GridItem(spacing: 16), count: columns), spacing: 16) {
+                ForEach(sponsors, id: \.self) { sponsor in
+                    SponsorItemView(sponsor: sponsor) {
+                        action(sponsor)
+                    }
+                }
+            }
+        }
+    }
+
+    func onTapItem(perform action: @escaping (Sponsor) -> Void) -> some View {
+        var view = self
+        view.action = action
+        return view
+    }
+}
+
+struct SponsorItemView: View {
+
+    let sponsor: Sponsor
+    let action: () -> Void
+
+    var body: some View {
+        ZStack {
+            AssetColors.white.swiftUIColor
+            Button {
+                action()
+            } label: {
+                NetworkImage(url: URL(string: sponsor.logo))
+                    .scaledToFit()
+                    .frame(minWidth: 0, maxWidth: .infinity)
+                    .frame(minHeight: 0, maxHeight: .infinity)
```

Comment:
I'm just curious this lines. 
Does specify `.infinity` to both argument isn't working?

```suggestion
                    .frame(minWidth: .infinity, maxWidth: .infinity)
```

Reply:
👀 
I did not know that! 

> I want to know the best practice to fill the parent view 🤔

I want to know too... Let's find out together 👍🏼